### DISCUSSION TITLE
What is ggml.org business plan ?

### DISCUSSION THREAD

**Discussion by ExtReMLapin** [ggml-org/llama.cpp#10194] [General]
I can't get my head around it.

The repository is active, pull requests, a lot of contributors, but what is the business plan behind it ? 

vllm gets resources from companies for testing and money from opencollective, what about llama.cpp/ggml.ai/org ?

Is there hidden sponsors ? 

What about a EuCB/Europol Inovation lab funding?

**Comment by WilliamTambellini** [comment]
http://ggml.ai/
?

**Reply by ExtReMLapin** [reply]
Having initial funds doesn't make it a business plan imo
